code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1452,6 +1452,7 @@
 		return
 	hide_underwear[UWC.name] = !hide_underwear[UWC.name]
 	update_underwear(1)
+	update_genitals_showing()

```

Comment:
Mark this line with a comment please, for future ease of resolving merge conflicts.